xml/security-barbican.xml

```diff
@@ -271,19 +246,101 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
     </para>
    </step>
   </procedure>
- </section>
- <section xml:id="sec.barbican.kmip">
-  <title>KMIP Plug-in Support</title>
+  <bridgehead renderas="sect4">Upgrade Master Key Configuration</bridgehead>
+  <procedure>
+   <step>
+    <para>
+     Check the master key.
+    </para>
+    <para>
+     If a master key is already defined, check
+     <filename>~/openstack/ardana/ansible/roles/barbican-common/vars/barbican_deploy_config.yml</filename>
+     for <literal>barbican_customer_master_key</literal> value. If the value
```

Comment:
better: `for the value of <literal>barbican_customer_master_key</literal>. If the value`